Control coverage/Feature/DBForMySql.md

```diff
@@ -0,0 +1,6 @@
+## DBForMySql
+
+| ControlId | Dependent Azure API(s) and Properties | Control spec-let |
+|-----------|-------------------------------------|------------------|
+| <b>ControlId:</b><br>Azure_DBforMySQL_Audit_Enable_ATP<br><b>DisplayName:</b><br>Enable Threat detection for MySQL database<br><b>Description: </b><br> Advanced Threat Protection must be enabled for Azure Database for MySQL |<b> ARM API to get security alert policy of a DBForMySql server: </b> <br> /subscriptions/{subscriptionId}<br>/resourceGroups/{resourceGroupName}<br>/providers/Microsoft.DBforMySQL<br>/servers/{serverName}<br>/securityAlertPolicies/Default<br>?api-version=2017-12-01 <br><b>Properties:</b><br> properties.state| <b>Passed: </b><br>ATP is enabled and 'email notifications to admins' are also enabled.<br><b>Failed: </b><br>Either MySQL is of 'basic tier' which does not support ATP. <br>Or ATP is disabled. <br>Or ATP is enabled but <br> 1. 'Email notifications to admins' is disabled as well as explicit email(s) are not configured. <br> 2. All 'Advanced threat protection types' are not enabled. |
```

Comment:
You may want to specify the other properties that are being considered for the evaluation - `emailAccountAdmins`, `emailAddresses`, `disabledAlerts`.